tests/test_scalars.py

```diff
@@ -0,0 +1,287 @@
+    check(assert_type(interval_f // 3, "pd.Interval[float]"), pd.Interval, float)
+    check(
+        assert_type(interval_td // 3, "pd.Interval[pd.Timedelta]"),
+        pd.Interval,
+        pd.Timedelta,
+    )
+
+    check(assert_type(interval_i // 3.5, "pd.Interval[float]"), pd.Interval, float)
+    check(assert_type(interval_f // 3.5, "pd.Interval[float]"), pd.Interval, float)
+    check(
+        assert_type(interval_td // 3.5, "pd.Interval[pd.Timedelta]"),
+        pd.Interval,
+        pd.Timedelta,
+    )
+
+    check(assert_type(interval_i - 1, "pd.Interval[int]"), pd.Interval, int)
+    check(assert_type(interval_f - 1, "pd.Interval[float]"), pd.Interval, float)
+    check(
+        assert_type(interval_ts - pd.Timedelta(days=1), "pd.Interval[pd.Timestamp]"),
+        pd.Interval,
+        pd.Timestamp,
+    )
+    check(
+        assert_type(interval_td - pd.Timedelta(days=1), "pd.Interval[pd.Timedelta]"),
+        pd.Interval,
+        pd.Timedelta,
+    )
+
+    check(assert_type(interval_i - 1.5, "pd.Interval[float]"), pd.Interval, float)
+    check(assert_type(interval_f - 1.5, "pd.Interval[float]"), pd.Interval, float)
+    check(
+        assert_type(interval_ts - pd.Timedelta(days=1), "pd.Interval[pd.Timestamp]"),
+        pd.Interval,
+    )
+    check(
+        assert_type(interval_td - pd.Timedelta(days=1), "pd.Interval[pd.Timedelta]"),
+        pd.Interval,
+    )
+
+    check(assert_type(interval_i + 1, "pd.Interval[int]"), pd.Interval)
+    check(assert_type(interval_f + 1, "pd.Interval[float]"), pd.Interval)
+    check(
+        assert_type(interval_ts + pd.Timedelta(days=1), "pd.Interval[pd.Timestamp]"),
+        pd.Interval,
+    )
+    check(
+        assert_type(interval_td + pd.Timedelta(days=1), "pd.Interval[pd.Timedelta]"),
+        pd.Interval,
+    )
+
+    check(assert_type(interval_i + 1.5, "pd.Interval[float]"), pd.Interval)
+    check(assert_type(interval_f + 1.5, "pd.Interval[float]"), pd.Interval)
+    check(
+        assert_type(interval_ts + pd.Timedelta(days=1), "pd.Interval[pd.Timestamp]"),
+        pd.Interval,
+    )
+    check(
+        assert_type(interval_td + pd.Timedelta(days=1), "pd.Interval[pd.Timedelta]"),
+        pd.Interval,
+    )
+
+    check(assert_type(0.5 in interval_i, bool), bool)
+    check(assert_type(1 in interval_i, bool), bool)
+    check(assert_type(1 in interval_f, bool), bool)
+    check(assert_type(pd.Timestamp("2000-1-1") in interval_ts, bool), bool)
+    check(assert_type(pd.Timedelta(days=1) in interval_td, bool), bool)
+
+    check(assert_type(hash(interval_i), int), int)
+    check(assert_type(hash(interval_f), int), int)
+    check(assert_type(hash(interval_ts), int), int)
+    check(assert_type(hash(interval_td), int), int)
+
+    interval_index_int = pd.IntervalIndex([interval_i])
+    interval_series_int = pd.Series(interval_index_int)
+
+    check(interval_series_int >= interval_i, pd.Series, bool)
```

Comment:
All of these tests from here to the bottom need `assert_type`

Reply:
This is fixed.